testing/scenario_app/android/run_gradle.py

```diff
@@ -0,0 +1,26 @@
+#!/usr/bin/env python
+# Copyright 2013 The Flutter Authors. All rights reserved.
+# Use of this source code is governed by a BSD-style license that can be
+# found in the LICENSE file.
+
+"""
+Invokes gradlew for building the scenario_app from GN/Ninja.
```

Comment:
does `gn` have a way to run a target? If yes, is this wrapper script still needed?

Reply:
GN knows how to run compiler binaries with some help, and it knows how to run python scripts. 

This is actually really hacky - we shouldn't be invoking gradle from GN, but refactoring that will take more work and we're no worse off than we were before invoking gradle from a shell script.

Reply:
does it know how to run a bash script?  If that's the case, then gradlew directly should work, no?

Reply:
That doesn't work on Windows.

Reply:
It could use gradlew.bat on Windows

Reply:
Which is all this python script is doing